### ISSUE TITLE
ErrorException Invalid argument supplied for foreach() 

### ISSUE THREAD

**Issue by jbov** [archtechx/tenancy#247]
#### Describe the bug
appears to be the same error as previously reported on a fresh Laravel 6 installation.
[2.2] ErrorException : Invalid argument supplied for foreach() TenancyServiceProvider.php:81#222
However in my case php artisan config:clear did not help.

#### Steps to reproduce

php artisan tenancy:install

   ErrorException  : Invalid argument supplied for foreach()

  at /Applications/MAMP/htdocs/ws2/vendor/stancl/tenancy/src/TenancyServiceProvider.php:34
    30|         $this->app->bind(Tenant::class, function ($app) {
    31|             return $app[TenantManager::class]->getTenant();
    32|         });
    33| 
  > 34|         foreach ($this->app['config']['tenancy.bootstrappers'] as $bootstrapper) {
    35|             $this->app->singleton($bootstrapper);
    36|         }
    37| 
    38|         $this->app->singleton(Commands\Migrate::class, function ($app) {

  Exception trace:

  1   Illuminate\Foundation\Bootstrap\HandleExceptions::handleError("Invalid argument supplied for foreach()", "/Applications/MAMP/htdocs/ws2/vendor/stancl/tenancy/src/TenancyServiceProvider.php", [])
      /Applications/MAMP/htdocs/ws2/vendor/stancl/tenancy/src/TenancyServiceProvider.php:34

  2   Stancl\Tenancy\TenancyServiceProvider::register()
      /Applications/MAMP/htdocs/ws2/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:609

  Please use the argument -v to see more details.

#### Your setup
 - Laravel version: [e.g. 6.2.0]
 - stancl/tenancy version: [e.g. 2.1.0]
 - Storage driver: [e.g. DB]

**Comment by stancl** [comment]
Could you share the repository with me?

**Comment by jbov** [comment]
https://github.com/jbov/ws2/invitations

**Comment by stancl** [comment]
```
git clone https://github.com/jbov/ws2
cd ws2
composer install
php artisan tenancy:install
```
works for me.

**Comment by jbov** [comment]
I just tried another fresh laravel installation and received the same 
error.  I believe you have an issue with the latest version of Laravel.

composer require stancl/tenancy
Using version ^2.2 for stancl/tenancy
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 1 install, 0 updates, 0 removals
   - Installing stancl/tenancy (v2.2.4): Loading from cache
Writing lock file
Generating optimized autoload files
 > Illuminate\Foundation\ComposerScripts::postAutoloadDump
 > @php artisan package:discover --ansi

    ErrorException  : Invalid argument supplied for foreach()

   at 
/Applications/MAMP/htdocs/ws2/vendor/stancl/tenancy/src/TenancyServiceProvider.php:34
     30|         $this->app->bind(Tenant::class, function ($app) {
     31|             return $app[TenantManager::class]->getTenant();
     32|         });
     33|
   > 34|         foreach ($this->app['config']['tenancy.bootstrappers'] 
as $bootstrapper) {
     35|             $this->app->singleton($bootstrapper);
     36|         }
     37|
     38| $this->app->singleton(Commands\Migrate::class, function ($app) {

   Exception trace:

   1 
Illuminate\Foundation\Bootstrap\HandleExceptions::handleError("Invalid 
argument supplied for foreach()", 
"/Applications/MAMP/htdocs/ws2/vendor/stancl/tenancy/src/TenancyServiceProvider.php", 
[])
/Applications/MAMP/htdocs/ws2/vendor/stancl/tenancy/src/TenancyServiceProvider.php:34

   2   Stancl\Tenancy\TenancyServiceProvider::register()
/Applications/MAMP/htdocs/ws2/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:609

   Please use the argument -v to see more details.
Script @php artisan package:discover --ansi handling the 
post-autoload-dump event returned with error code 1

**Comment by stancl** [comment]
Works for me on 6.7.0, even after running `composer update`.

**Comment by antonio-aguirre** [comment]
Hi, i have the same error. can you help me @stancl  please 

I did these steps:

-First I wrote in command line **composer require stancl/tenancy** but appear the same issues mentioned at the top.

- next I wrote **composer update** for remove the stancl/tenancy package, I tried again to run the  **composer require stancl/tenancy** command but error appear again.

I don't know why, I installed the same package with the same Laravel  version (6.18.3) in another project and this have no problem.

My GitHub project: https://github.com/antonio-aguirre/FAQ-ZOOM.git

**Comment by stancl** [comment]
Show me the stack trace.

**Comment by antonio-aguirre** [comment]
> Show me the stack trace.

MacBook-Pro-de-Antonio:FAQ-Zoom antonioaguirre$ composer require stancl/tenancy
Using version ^2.3 for stancl/tenancy
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 1 install, 0 updates, 0 removals
  - Installing stancl/tenancy (v2.3.1): Loading from cache
Package jakub-onderka/php-console-color is abandoned, you should avoid using it. Use php-parallel-lint/php-console-color instead.
Package jakub-onderka/php-console-highlighter is abandoned, you should avoid using it. Use php-parallel-lint/php-console-highlighter instead.
Writing lock file
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi

   ErrorException  : Invalid argument supplied for foreach()

  at /Users/antonioaguirre/Sites/FAQ-Zoom/vendor/stancl/tenancy/src/TenancyServiceProvider.php:34
    30|         $this->app->bind(Tenant::class, function ($app) {
    31|             return $app[TenantManager::class]->getTenant();
    32|         });
    33| 
  > 34|         foreach ($this->app['config']['tenancy.bootstrappers'] as $bootstrapper) {
    35|             $this->app->singleton($bootstrapper);
    36|         }
    37| 
    38|         $this->app->singleton(Commands\Migrate::class, function ($app) {

  Exception trace:

  1   Illuminate\Foundation\Bootstrap\HandleExceptions::handleError("Invalid argument supplied for foreach()", "/Users/antonioaguirre/Sites/FAQ-Zoom/vendor/stancl/tenancy/src/TenancyServiceProvider.php", [])
      /Users/antonioaguirre/Sites/FAQ-Zoom/vendor/stancl/tenancy/src/TenancyServiceProvider.php:34

  2   Stancl\Tenancy\TenancyServiceProvider::register()
      /Users/antonioaguirre/Sites/FAQ-Zoom/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:616

  Please use the argument -v to see more details.
Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1

Installation failed, reverting ./composer.json to its original content.

**Comment by stancl** [comment]
Did you run this? Do you have a `tenancy` config file already in your config directory?
```
php artisan clear
```

**Comment by stancl** [comment]
Also note that if you ran `php artisan tenancy:install`, you can't just re-install the package using that command due to the changes it did to your Http Kernel.

**Comment by antonio-aguirre** [comment]
> Did you run this? Do you have a `tenancy` config file already in your config directory?
> 
> ```
> php artisan clear
> ```

I have a new project, only with minor modifications, and all this happened when I tried to install the package.

I ran the php artisan clear command but nothing happened.

but right now I ran the package installation command again and nothing bad happened anymore ... this was weird but I'm calm for now.

I must say thank you for your time to respond, and your package has helped me a lot in another project!
